dev_guide/templates.adoc

```diff
@@ -51,41 +51,22 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
 
-There is also the ability to add labels in the template from the command line.
+You can create a configuration from a template with either the Management Console or the CLI, after first uploading a template to your OpenShift instance.
 
-****
-`$ osc process -f _<filename>_ -l name=otherLabel`
-****
+Create a template json file, like the above example, then upload it with the the CLI using the following process:
 
-== Parameters
-The list of parameters that you can override are listed in the `*parameters*` section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+=== Uploading a Template
+Load a json template file into the OpenShift template library with the following command:
```

Comment:
Just an effort to link back to an example of what the JSON could look like, for convenience. And the "Overview" happened to be where the example is. If you don't like that, you could also create your own custom anchor right above the example block, by adding something like:

```
[[sample-template-json-file]]
```

and later linking to that:

```
by passing link:#sample-template-json-file[a JSON file] to the following command:
```